core/domain/feedback_services.py

```diff
@@ -504,25 +503,14 @@ def get_thread_summaries(user_id, thread_ids):
                 second_last_message_read = (
                     last_two_messages[index][1].message_id
                     in feedback_thread_user_models[index].message_ids_read_by_user) # pylint: disable=line-too-long
-            else:
-                second_last_message_read = False
 
-            if last_two_messages[index][1].author_id is None:
-                author_second_last_message = None
-            else:
+            if last_two_messages[index][1].author_id is not None:
                 author_second_last_message = user_services.get_username(
                     last_two_messages[index][1].author_id)
         if not last_message_read:
             number_of_unread_threads += 1
 
-        if thread.message_count:
-            total_message_count = thread.message_count
-        # TODO(Arunabh): Remove else clause after each thread has a message
-        # count.
-        else:
-            total_message_count = (
-                feedback_models.GeneralFeedbackMessageModel
-                .get_message_count(thread_ids[index]))
+        total_message_count = thread.message_count
```

Comment:
I deleted above conditions as message count gets updated above in:
https://github.com/oppia/oppia/blob/0b704b885d000fe70c15ea3226a955af15310496/core/domain/feedback_services.py#L405

Reply:
Thanks for doing this. I had thought this was removed before. Yeah, I am fine with this.